pipeline/scripts/7-collectSamplesGroupedHMDB.R

```diff
@@ -67,7 +67,7 @@ if (n>=sub & (floor(n/sub)-1)>=2){
       n_moved = 0
       
       # Calculate 3ppm and replace border, avoid cut within peakgroup!
```

Comment:
Comment aanpassen, is geen 3ppm meer.

Reply:
dat gaan we een keer in een grote "code cleanup" actie doen. Dit zijn namelijk niet de enige comments op deze manier, want hiervoor was het ook geen 3ppm ;). ook willen we extra comments toevoegen. de comments die er nu staan klopte bij de vorige versie dus al niet meer.